R/random-data-generation.R

```diff
@@ -0,0 +1,54 @@
+library(openxlsx)
+
+prenoms <- c("Hector","Mari","Amance","Peter","Nathalie","Agathe","Romain","Eve","Marc","Enzo")
+noms <- c("Dupont","Nguyen","Martin","Garcia","Fouquier","Dubois","Jack","Boisu","Zaky","Mathy")
+filiere <- c(rep("EMIR",30),rep("MICA",30),rep("CLASSIQUE",140))
+
+#réponses possibles
+reponse_classique <- c("EII","MA","INFO","ET","GPM","GMA","GCU")
+reponse_emir <- c("EII","INFO","GPM","ET")
+reponse_mica <- c("GCU","MA","GMA","INFO")
+
+choix_colonnes <- c(
+  paste0("Q02_Voeux->", reponse_classique),
+  paste0("Q03_VoeuxEMIR->", reponse_emir),
+  paste0("Q04_voeuxMICA->", reponse_mica)
+)
+
+nb_etudiants <- 200
+
+df <- data.frame(
+  prenom = sample(prenoms, nb_etudiants, replace = TRUE),
+  nom = sample(noms, nb_etudiants, replace = TRUE),
+  filiere = sample(filiere, nb_etudiants, replace = TRUE)
+)
+
+generer_voeux <- function(filiere) {
```

Comment:
Created functions should be documented using the roxigen2 standard.
You can read `R/input.R` on the `dev` branch, or [this page](https://kbroman.org/pkg_primer/pages/docs.html) to see how roxygen2 works.
After adding documentation, you can run the command: `devtools::document()`.